app/views/discovery_rules/_form.html.erb

```diff
@@ -16,7 +16,7 @@
           <div class="tab-pane active" id="primary">
             <%= text_f f, :name %>
             <%= autocomplete_f f, :search,
-                               :path             => "discovered_hosts",
+                               :path             => "/discovered_hosts",
```

Comment:
Is this related? Can you explain?

Reply:
Yes, this is what lead to 404 error.

This form used to get rendered inside the `two-pane` view, which keep the index page url a the browser) and now it get rendered inside it own page with it own url `discovery_rules/new` or `discovery_rules/:id/edit`.
It makes the url wrong:
http://localhost:3000/discovery_rules/discovered_hosts/auto_complete_search?search=

Reply:
I migrated it to use `discovered_hosts_path` :+1: